### PR TITLE
Create WeGif

### PR DESCRIPTION
**Pull request by zhupengxiang**
WeGif：The main function of WeGIF is to upload and search GIF images, emoticons, etc. Mass storage for easy query

### CODE REVIEW

**Comment by zhupengxiang** [comment]
When can the audit be completed

**Review by zhupengxiang — COMMENTED** [review]
WeGif

File: participants/WeGif [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below. 
+We collect data sets on the Internet by ourselves. 
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition? 
+1T - 10T 
+## (7) How are you structuring the data? 
+Structured .zip file of the dataset 
+## (8) What pre-processing are you doing before ingesting the data? 
+*We will directly deposit data into the network through miner transactions* 
+## (9) What tech stack will you use to store the data? 
+lotus
```
Comment (dineshshenoy) [inline participants/WeGif:21]: You may want to consider IPFS or Powergate for hot retrieval.

**Review by dineshshenoy — APPROVED** [review]

**Comment by zhupengxiang** [comment]
When can I check the competition rankings？ thanks

**Comment by dineshshenoy** [comment]
It should be posted within 24 hours.

**Pull request merged**